### ISSUE TITLE
Subtitle extracting files with KeyError: 'segs' for subtitle events describing environment cues

### ISSUE THREAD

**Issue by KoushiroMitani** [tubearchivist/tubearchivist#249]
I keep hitting this stack trace which is causing the download process to halt. If I restart downloads again it will eventually proceed on.
I've noticed this keeps happening with the same specific channel. 

```
[2022-05-17 11:44:24,587: WARNING/ForkPoolWorker-32] nsIi5IDmALI-en: get user uploaded subtitles
[2022-05-17 11:44:24,788: ERROR/ForkPoolWorker-32] Task download_pending[e6df8867-7059-454d-b809-83d6a1594f0a] raised unexpected: KeyError('segs')
Traceback (most recent call last):
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 451, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/root/.local/lib/python3.10/site-packages/celery/app/trace.py", line 734, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/home/tasks.py", line 89, in download_pending
    downloader.run_queue()
  File "/app/home/src/download/yt_dlp_handler.py", line 182, in run_queue
    vid_dict = index_new_video(
  File "/app/home/src/index/video.py", line 628, in index_new_video
    video.check_subtitles()
  File "/app/home/src/index/video.py", line 612, in check_subtitles
    handler.download_subtitles(relevant_subtitles=subtitles)
  File "/app/home/src/index/video.py", line 134, in download_subtitles
    parser.process()
  File "/app/home/src/index/video.py", line 200, in process
    "text": "".join([i.get("utf8") for i in event["segs"]]),
KeyError: 'segs'
```

**Comment by bbilly1** [comment]
That video doesn't look like it exists any more. Or is it available for you?

**Comment by KoushiroMitani** [comment]
It's still available to me: https://www.youtube.com/watch?v=nsIi5IDmALI  This has been cropping up for me with numerous videos on the PBS Frontline channel.

**Comment by bbilly1** [comment]
yt-dlp is the deciding factor here:

```
❯ yt-dlp 'https://www.youtube.com/watch?v=nsIi5IDmALI' --simulate
[youtube] nsIi5IDmALI: Downloading webpage
[youtube] nsIi5IDmALI: Downloading android player API JSON
ERROR: [youtube] nsIi5IDmALI: Video unavailable. This video is not available
```

If yt-dlp can't extract the metadata Tube Archivist won't be able to either, that's a [known limitation](https://github.com/tubearchivist/tubearchivist#known-limitations).

**Comment by bbilly1** [comment]
From our Discord debugging: Some environment describing subtitle events won't have a *segs* key, this can be safely skipped. Needs error handling.

**Comment by PhuriousGeorge** [comment]
Great debugging! I've ran into this myself on a few videos but haven't gotten around to asking about it yet, thanks @KoushiroMitani & @bbilly1

**Comment by bbilly1** [comment]
Thanks for your help, fix is now published in v0.1.6, please update.
